build.gradle.kts

```diff
@@ -104,7 +104,7 @@ allprojects {
                 implementation(deps.clojure.lang)
 
                 testRuntimeOnly("dev.clojurephant", "jovial", "0.4.1")
-                nrepl("cider", "cider-nrepl", "0.47.1")
+                nrepl("cider", "cider-nrepl", "0.50.1")
```

Comment:
this commit can go direct to `main`, dw about including these in a PR (see git.adoc)